.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: ruby
 
 rvm:
-  - 2.1.2
+  - 2.2.0
+  - 2.1.4
   - 2.0.0
   - 1.9.3
   - ruby-head
-  - jruby-19mode
+  - jruby-1.7.17
```

Comment:
Because 1.7.18 is broken on Travis, too. It uses the `fuzzy` parameter and results in grabbing JRuby 9000. Specifying 1.7.17 is a [recommended workaround](https://github.com/travis-ci/travis-ci/issues/3067#issuecomment-68293804).